README.md

```diff
@@ -79,6 +80,10 @@ Login to your Investimental account and click on the "Orders Daily Log". Select
 
 ![Export instructions for Investimental](./assets/export-investimental.png)
 
+### Parqet
+
+todo
```

Comment:
Instructions suggestions: 

Login to Parquet and avigate to the 'Activities' section (in German, 'Aktivitäten'). In the top-right corner, next to the green 'Neue Aktivität' button, you'll see an option to 'Download as CSV.' Clicking on this will download a CSV file containing all your activities.

Reply:
Added this to the instructions